12.0 lcc members v3 portal registration

stephansainleger opened the pull request:
New addons **lcc_members_portal** to allow portal users to register themselves as member to the LCC,

Commit:
[NEW] Add new fields on partner model for newsletter and policy

Commit:
[NEW] New addon lcc_member_portal

with first process working: private registration process

njeudy reviewed (commented):
Ok here his my test:
- install lcc_members_portal module
- create a portal user
- A create an membership product that is check 'use for portal membeship'
- login with and test.

and test :)

njeudy committed:
[FIX] fixe missing impoort and error message

njeudy committed:
[FIX] add some required and missing field:
- email (missing and required)
- phone (required)
- country (required)
this fields are required in ecommerce process
